### DISCUSSION TITLE
How does scoring work?

### DISCUSSION THREAD

**Discussion by mciureanu** [Citolab/qti-components#3] [General]
Hello and thank you for this project. I was trying to evaluate it, but I'm stuck with getting the score. I guess this is not implemented?

**Comment by herrKlein** [comment]
Hi, scoring should work. The way it works you should call processResponse() method on the current QtiAssessmentItem in the dom. then the scoring rules will be applied and an event will be thrown for every qti variable that changes

**Reply by herrKlein** [reply]
the event is called "qti-outcome-changed"

**Comment by mciureanu** [comment]
Thank you, it works indeed!
